eduNEXT/xblock-mindmap
# fix: allow to see multiple xblock mindmap in lms

**Description**
When we create more than one XBlock per unit  in the LMS, it breaks due to a draggable plugin from jsmind that was initialized previously, which is weird because each XBlock in the LMS is an iframe. This fix allows us to create any number of XBlocks  without that issue



BEFORE           | NOW
:-------------------------:|:-------------------------:
![](https://github.com/eduNEXT/xblock-mindmap/assets/134975835/36bdbdb8-05d8-48e9-932e-dad5ee7a207b)  |  ![](https://github.com/eduNEXT/xblock-mindmap/assets/134975835/4b0bc814-917a-42b0-9924-6b60b23960d3)


### How to test 

1. Once you have the XBlock in your environment, go to Studio > create a unit > create some XBlock-type mind maps.
2. Publish the changes.
3. Go to the LMS to see the unit.



**Merge checklist:**
Check off if complete *or* not applicable:
